src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -188,6 +191,77 @@ func (c controller) PostInstallConfigs(wg *sync.WaitGroup) {
 	c.sendCompleteInstallation(true, "")
 }
 
+func (c controller) UpdateBMHs(wg *sync.WaitGroup) {
+	defer wg.Done()
+	for {
+		time.Sleep(GeneralWaitTimeout)
+		exists, err := c.kc.IsMetalProvisioningExists()
```

Comment:
it needs to be check more than once?

Reply:
i prefer to check it every loop, since in case there is a network gleech , i won't accidentally start updating BMHs.

Reply:
fixed